src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -113,6 +126,78 @@ def _try_fetch(self) -> Iterable[ExternalLocation]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield ExternalLocation(*row)
 
+    def _get_ext_location_definitions(self, missing_locations: list[ExternalLocation]) -> list:
+        tf_script = []
+        cnt = 1
+        for loc in missing_locations:
+            if loc.location.startswith("s3://"):
+                res_name = loc.location[5:].replace("/", "_")[:-1]
+            elif loc.location.startswith("gcs://"):
+                res_name = loc.location[6:].replace("/", "_")[:-1]
+            elif loc.location.startswith("abfss://"):
+                res_name = (
+                    loc.location[loc.location.index("@") + 1 :]
+                    .replace(".dfs.core.windows.net", "")
+                    .replace("/", "_")[:-1]
+                )
+            else:
+                # if the cloud storage url doesn't match the above condition or incorrect (example wasb://)
+                # generate a dummy resource name
```

Comment:
Wasb is not supported by UC ext locations. Raise this as warning and don't include this location in the list of tf resources